src/lib/fragments/AgGrid.react.js

```diff
@@ -528,6 +539,43 @@ export default class DashAgGrid extends Component {
         });
 
         this.updateColumnWidths();
+
+        if (!isEmpty(filterModel)) {
+            this.state.gridApi.setFilterModel(filterModel);
+        }
+
+        if (resetColumnState) {
+            this.resetColumnState(false);
+        }
+
+        if (exportDataAsCsv) {
+            this.exportDataAsCsv(csvExportParams, false);
+        }
+
+        if (selectAll) {
+            this.selectAll(selectAll, false);
+        }
+
+        if (deselectAll) {
+            this.deselectAll(false);
+        }
+
+        if (autoSizeAllColumns) {
+            this.autoSizeAllColumns(autoSizeAllColumns, false);
+        }
+
+        if (deleteSelectedRows) {
+            this.deleteSelectedRows(false);
+        }
+
+        setProps({
+            exportDataAsCsv: false,
+            selectAll: false,
+            deselectAll: false,
+            autoSizeAllColumns: false,
+            deleteSelectedRows: false,
+        });
```

Comment:
Also this one - we should only `setProps` the ones that weren't already `false`.

Reply:
Alright, I'm using the same way the above works. `propsToSet` then build through each, then if `propsToSet` not empty, it `setProps`